data-sources/conviva/config.yml

```
@@ -0,0 +1,11 @@
+id: conviva
+displayName: NRI Conviva Integration
```

I don't think we want to include the word "Integration" since _all_ of these quickstarts are technically "integrations". Feels a bit repetitive.

```suggestion
displayName: NRI Conviva
```

---

Also, what does `NRI` mean? This might not be very clear to the uninitiated (I too don't know what it means). Perhaps we should spell it out.

Furthermore, if the `NR` part is "New Relic"...we might not _want_ that in the title. All of these are _New Relic_ quickstarts, so we don't need to qualify this quickstart with that extra title.